tests/consume.el

```diff
@@ -0,0 +1,59 @@
+;;; tests/consume.el --- Some tests for js2-mode.
+
+;; Copyright (C) 2009, 2011-2016  Free Software Foundation, Inc.
```

Comment:
Could you fix the dates? These should only list the years the file has seen any changes. Just 2016 in this case, IOW.